autosubmit/monitor/monitor.py

```diff
@@ -253,14 +302,14 @@ def create_tree_list(self, expid, joblist, packages, groups, hide_groups=False):
                         if group_name not in graph.obj_dict['subgraphs']:
                             graph.add_subgraph(subgraph)
             else:
-                for edge in copy.deepcopy(exp.obj_dict['edges']):
+                for edge in deepcopy(exp.obj_dict['edges']):
```

Comment:
Whenever possible I imported the functions used instead of the whole module, as in the memory optimization work we did recently the imports were bringing a lot of junk never used.